checklists/checklist.php

```diff
@@ -662,7 +662,7 @@ function changeImageSource(elem){
 								//Edit family name display style here
 								?>
 								<div class="family-div" id="<?php echo strip_tags($group);?>">
-									<a href="<?php echo htmlspecialchars($famUrl, HTML_SPECIAL_CHARS_FLAGS); ?>" target="_blank" style="color:black;"><?php echo htmlspecialchars($group, HTML_SPECIAL_CHARS_FLAGS);?></a>
+									<a href="<?php echo strip_tags($famUrl, HTML_SPECIAL_CHARS_FLAGS); ?>" target="_blank" style="color:black;"><?php echo strip_tags($group, HTML_SPECIAL_CHARS_FLAGS);?></a>
```

Comment:
Note that `strip_tags` doesn't take the same second argument that `htmlspecialchars` does. Please remove the `HTML_SPECIAL_CHARS_FLAGS` argument. One argument is fine for PHP 8.0+